src/icons/fileIcons.ts

```diff
@@ -1757,6 +1757,14 @@ export const fileIcons: FileIcons = {
       fileExtensions: ['svelte'],
       fileNames: ['svelte.config.js', 'svelte.config.cjs'],
     },
+    {
+      name: 'javascript-svelte',
+      fileNames: ['+page.js', '+page.server.js', '+layout.js', '+server.js'],
```

Comment:
I was wondering what the "+" means? 🧐

Reply:
All SvelteKit specific files start with a “+” to distinct them from other files in the same directory (e. g. custom Components)

Reply:
https://kit.svelte.dev/docs/routing

> Each route directory contains one or more _route files_, which can be identified by their `+` prefix.

Reply:
I’ll resolve this conversation